src/assemble_workflow/bundle_recorder.py

```diff
@@ -44,7 +41,7 @@ def __get_package_name(self, build: BuildManifest.Build) -> str:
             build.platform,
             build.architecture,
         ]
-        extension = self.EXTENSIONS[self.distribution] if self.distribution else self.EXTENSIONS['tar']
+        extension = Dists().DISTRIBUTIONS_MAP[self.distribution][1] if self.distribution else Dists().DISTRIBUTIONS_MAP['tar'][1]
```

Comment:
This creates a new instance of `Dists()`, which shouldn't be necessary since it's a constant. `Dists.DISTRIBUTION_MAPS` should work.

Reply:
Will try, thanks.